app/javascript/src/components/Dashboard/Contacts/Pane/CreateContact.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui/v2";
+
+import formInitialValues from "constants/formInitialValues";
+
+import Form from "./Form";
+
+export default function NewContactPane({ fetchContacts, showPane, setShowPane }) {
```

Comment:
Please refrain from the suffix `Pane`.
Also, please move `initialValues` and `validationSchema` for the form to the scope of this directory.

Reply:
Also, please nix out the folder `Pane`.